appcircle_publish_testflight_status_check/1.1.1/component.yaml

```diff
@@ -0,0 +1,29 @@
+platform: iOS
+purpose: Publish
+structureType: CustomUI
+buildPlatform:
+displayName: "Get Approval from TestFlight"
+description: "Get approval for the publish from TestFlight"
+docsUrl: https://docs.appcircle.io/publish-integrations/ios-publish-integrations/approval-test-flight
+requiredComponents: "appcircle_publish_send_to_testflight"
+precedingComponents: "appcircle_publish_appstore_app_information"
```

Comment:
Why we need to add this step after the appcircle_publish_appstore_app_information step?